app/assets/stylesheets/media.scss

```diff
@@ -101,3 +102,32 @@
   }
 
 }
+
+// VideoJS CSS overrides for responsive layout
+// See http://andreassauer.name/experimente/video-js/demo.html
+.video-js {
+  width: 100% !important;
```

Comment:
Given that we're overriding the upstream css, this seems to be required (as indicated by the link mentioned in the comment).